Description: Detect the webroot to be used based on the distro.
Author: Chuck Short <zulcss@ubuntu.com>
Upstream: No
diff -Naurp cobbler-2.1.0.orig//setup.py cobbler-2.1.0//setup.py
--- cobbler-2.1.0.orig//setup.py	2011-01-28 10:23:01.000000000 -0500
+++ cobbler-2.1.0//setup.py	2011-01-28 11:01:44.226003787 -0500
@@ -128,7 +128,13 @@ if __name__ == "__main__":
     libpath     = "/var/lib/cobbler/"
     logpath     = "/var/log/"
 
-    webroot     = "/var/www/"
+    if os.path.exists("/etc/SuSE-release"):
+        webroot = "/srv/www/"
+    elif os.path.exists("/etc/debian_version"):
+        webroot = "/usr/share/cobbler/webroot/"
+    else:
+        webroot = "/var/www/"
+
     if os.path.exists("/etc/SuSE-release"):
         webconfig  = "/etc/apache2/conf.d"
     elif os.path.exists("/etc/debian_version"):
